media-plugins/swh-lv2/swh-lv2-1.0.16.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Large collection of LV2 audio plugins/effects"
+HOMEPAGE="http://plugin.org.uk/"
+SRC_URI="https://github.com/swh/lv2/archive/v${PV}.tar.gz"
```

Comment:
Please rename it: v${PV}.tar.gz -> ${P}.tar.gz

Since our distfiles end up in the same dir, if all package tarballs were named version.tar.gz we'd have a lot of collisions